frameworks/template/README.md

```diff
@@ -396,58 +387,50 @@ In case of problems with accessing the DC/OS Dashboard, logs may also be accesse
 1. You should now see two lists of tasks. `Active Tasks` are what's currently running, and `Completed Tasks` are what has since exited. Click on the `Sandbox` link for the task you wish to examine.
 1. The `Sandbox` view will list files named `stdout` and `stderr`. Click the file names to view the files in the browser, or click `Download` to download them to your system for local examination. Note that very old tasks will have their Sandbox automatically deleted to limit disk space usage.
 
-<a name="knownissues"></a>
-# Known Issues
-
-- _LIST ANY KNOWN BUGS OR ISSUES WITH YOUR SERVICE INTEGRATION (AND THEIR WORKAROUNDS) HERE_
 
 <a name="limitations"></a>
 # Limitations
 
-- _LIST SUMMARY OF CAVEATS OR USEFUL KNOWLEDGE FOR RUNNING THE SERVICE HERE_
-- Shrinking cluster size (number of nodes) is not supported.
-
-## Removing a Node
-
-Removing a node is not supported at this time.
-
-## Automatic Failed Node Recovery
-
-Nodes are not automatically replaced by the service in the event a system goes down. You may either manually replace nodes as described under [Managing](#managing), or build your own ruleset and automation to perform this operation automatically.
+_MANAGE CUSTOMER EXPECTIONS BY DISCLOSING ANY FEATURES OF YOUR PRODUCT THAT ARE NOT SUPPORTED WITH DC/OS, FEATURES MISSING FROM THE DC/OS INTEGRATION, ETC._
 
 ## Updating Storage Volumes
 
 Neither volume type nor volume size requirements may be changed after initial deployment.
 
 ## Rack-aware Replication
 
-Rack awareness within the service is not currently supported, but is planned to be supported with a future release of DC/OS.
+Rack awareness placement and awareness is not support at this time.
 
-## _ANY OTHER CAVEATS TO MENTION HERE?
-
-_FOR EXAMPLE, DOES YOUR SERVICE REQUIRE MANUAL INVOLVEMENT BY THE USER IN CERTAIN SCENARIOS?_
+## Removing a Node
 
-<a name="terms"></a>
-# Terms of Use
+Removing a node is not supported at this time.
 
-_ANY RISK WARNINGS OR REQUIREMENTS FOR SUPPORTED ENVIRONMENTS GO HERE_
+## _OTHER CAVEATS SPECIFIC TO YOUR PRODUCT INTEGRATION_
 
 <a name="support"></a>
 # Support
 
-## Supported Versions
+## Package Versioning Scheme
 
-- _SERVICENAME_: _WHAT VERSION OF YOUR SERVICE IS INCLUDED IN THE PACKAGE?_
-- DC/OS: _LIST VERSION(S) OF DC/OS THAT YOU'VE TESTED AND SUPPORT_
+Packages are versioned with an `a.b.c-x.y.z` format, where `a.b.c` is the version of the DC/OS integrtion and `x.y.z` indicates the version of _SERVICENAME_. For example, `1.5.0-3.2.1` indicates version `1.5.0` of the DC/OS integrtion and version `3.2.1` of _SERVICENAME_.
 
-## Package Versioning
+## Contacting Technical Support
 
-Packages are versioned with an `a.b.c-x.y.z` format, where `a.b.c` is the version of the service management layer and `x.y.z` indicates the version of _SERVICENAME_. For example, `1.5.0-3.2.1` indicates version `1.5.0` of the service management layer and version `3.2.1` of _SERVICENAME_.
```

Comment:
I'd recommend leaving this example text here.  It would be good to guide the partner toward the proper format so they're not surprised in the certification process.